main.tf

```diff
@@ -143,6 +150,11 @@ resource "kubernetes_deployment" "wandb" {
             }
           }
 
+          env {
+            name  = "TERRAFORM_VERSION"
+            value = tostring(random_integer.number.result)
```

Comment:
Instead of a random integer we should be able to use the actual module version here.  @jsbroks how can we get the  automated release process to update a variable that is set to the current version of the TF module?  Ideally we could do something like `module.this.version` here but I think that means we need to define a version output...